app/services/reports/querier.rb

```diff
@@ -17,12 +17,20 @@ def perform
       OrderDetailListTransformerFactory.instance(order_details).perform
     end
 
-    def order_details
+    def order_details(options = {})
```

Comment:
Do you know what branch that is? I don't see where this method is being called publicly anywhere.